test/commands/lib/import.test.js

```diff
@@ -277,7 +277,7 @@ test('enrich $ims jwt credential with project.org.ims_org_id', async () => {
   expect(fs.writeFile).toHaveBeenCalledTimes(2)
 })
 
-test('do not enrich $ims.jwt with ims_org_id if no jwt credentials defined ', async () => {
+test('do not enrich ims.jwt with ims_org_id if no jwt credentials defined ', async () => {
```

Comment:
nitpick: ims.contexts.jwt (or make it consistent with the test above :)

Reply:
ok makes sense